### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by paulmelnikow**

### CODE REVIEW

File: start-shields.sh [hunk 1]
```diff
@@ -5,7 +5,5 @@ SHIELDS_DIR="$DIR/shields"
 cd "$SHIELDS_DIR"
 export HTTPS=true
 export SHIELDS_ANALYTICS_FILE="$SHIELDS_DIR"/analytics-https.json
-export ALLOWED_ORIGIN=http://shields.io,https://shields.io
-export REDIRECT_URI=https://shields.io/
-export GITHUB_DEBUG_ENABLED=false
```
Comment (paulmelnikow) [inline start-shields.sh:10]: This is the default.

File: start-shields.sh [hunk 2]
```diff
@@ -5,7 +5,5 @@ SHIELDS_DIR="$DIR/shields"
 cd "$SHIELDS_DIR"
 export HTTPS=true
 export SHIELDS_ANALYTICS_FILE="$SHIELDS_DIR"/analytics-https.json
-export ALLOWED_ORIGIN=http://shields.io,https://shields.io
-export REDIRECT_URI=https://shields.io/
```
Comment (paulmelnikow) [inline start-shields.sh:9]: These config values are now checked into the main repo.

**Review by espadrine — APPROVED** [review]
I'm glad we can remove those! The less we have here, the better, as this script is not deployable automatically.

Hopefully, as we discussed on chat, we can remove NODE_CONFIG_ENV in the future.

**Comment by paulmelnikow** [comment]
> I'm glad we can remove those! The less we have here, the better, as this script is not deployable automatically.

Agreed! This is a big improvement in that regard.

**Pull request merged**